### PR TITLE
[RDY] Moved prombench to subdirectory prombench

### PR DESCRIPTION
**Pull request by geekodour**
See #257

### CODE REVIEW

**Comment by geekodour** [comment]
cc @VladMasarik

**Review by krasi-georgiev — COMMENTED** [review]
We need a readme file in the root explaining about both projects.
I think @VladMasarik 	already had this(it is removed now) so look through the PR commits

**Review by krasi-georgiev — COMMENTED** [review]
We need a readme file in the root explaining about both projects.
I think @VladMasarik 	already had this(it is removed now) so look through the PR commits

**Comment by geekodour** [comment]
@krasi-georgiev `make build` is working and docker images are getting built, why do we need to have a separate `Makefile` for every tool then?

File: prombench/.gitignore [hunk 1]
```diff
@@ -1,5 +1 @@
 /prombench
```
Comment (krasi-georgiev) [inline prombench/.gitignore:1]: Is the gitignore really working from a subdir?
`prombench/.gitignore`

Comment (geekodour) [reply prombench/.gitignore:1]: Yes, it's working.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
 # test-infra
```
Comment (krasi-georgiev) [inline README.md]: don't think it is needed, can make the next line a small title.
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,125 +1,18 @@
-# Automated Prometheus E2E testing and benchmarking.
+# test-infra
+This repository contains tools and configuration files for the testing and benchmarking used in the Prometheus project.
```
Comment (krasi-georgiev) [inline README.md]:
```suggestion
## This repository contains tools and configuration files for the testing and benchmarking used in the Prometheus project.
```

File: README.md [hunk 4]
```diff
@@ -1,125 +1,18 @@
...
-    -v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
-    -f manifests/prombench/benchmark
-```
-
-## Buliding from source
-To build Prombench and related tools from source you need to have a working Go environment with go modules enabled. Prombench uses [promu](https://github.com/prometheus/promu) to build the binaries.
-```
-make build
-```
+## Buliding test-infra tools from source
```
Comment (krasi-georgiev) [inline README.md]:
```suggestion
## Buliding from source
```

**Review by krasi-georgiev — COMMENTED** [review]
sorry missed these in the first review

**Pull request merged**